chain/rosetta-rpc/src/adapters/mod.rs

```diff
@@ -410,6 +410,7 @@ impl From<NearActions> for Vec<crate::models::Operation> {
                     );
                     operations.push(deploy_contract_operation);
                 }
+                near_primitives::transaction::Action::Delegate(_) => todo!(),
```

Comment:
Should this `todo` be fixed for commiting this patch or it can be done in another patch?

Reply:
I think we currently don't have a way to parse `DelegateAction` without the feature enabled. So it would only be possible to panic on nightly builds. But if you want to fix this before we merge, @firatNEAR  told me they will take care of this one.